webapp/src/action_types/index.js

```diff
@@ -1,9 +1,13 @@
+import {id as pluginId} from '../manifest';
+
 export default {
-    RECEIVED_REVIEWS: 'received_reviews',
-    RECEIVED_YOUR_PRS: 'received_your_prs',
-    RECEIVED_YOUR_ASSIGNMENTS: 'received_your_assignments',
-    RECEIVED_MENTIONS: 'received_mentions',
-    RECEIVED_UNREADS: 'received_unreads',
-    RECEIVED_CONNECTED: 'received_connected',
-    RECEIVED_GITHUB_USER: 'received_github_user',
+    RECEIVED_REVIEWS: pluginId + '_received_reviews',
```

Comment:
Adding the pluginId to the action name is a convention recommended for plugins to avoid collision of redux actions in the webapp as suggested here: https://github.com/mattermost/mattermost-plugin-demo/blob/master/webapp/src/action_types.js